crowbar_framework/app/models/api/storage.rb

```diff
@@ -15,6 +15,32 @@
 #
 
 module Api
-  class Storages
+  class Storage < Tableless
+    def repocheck
+      {}.tap do |ret|
+        ret["ceph"] = {}
+        ceph_node_architectures = NodeObject.find(
+          "roles:ceph-mon AND ceph_config_environment:*"
```

Comment:
It will very likely not happen for ceph, but for openstack, it's different, and since I assume there'll be some copy & paste, I think it's better to keep the generic code :-)